schema/schema.go

```diff
@@ -50,6 +51,16 @@ type Column struct {
 	Ignored Ignored
 }
 
+// Fkey respresents a foreign key.
+type Fkey struct {
+	Column      []string
```

Comment:
Maybe should be Columns i.e. plural.
Since it is a list of Columns.
Same for ReferColumns.

Reply:
Done